setup/migrate.py

```diff
@@ -23,11 +23,11 @@ def move_file(fn, domain_name_escaped, filename):
 	# Migrate the 'domains' directory.
 	for sslfn in glob.glob(os.path.join( env["STORAGE_ROOT"], 'ssl/domains/*' )):
 		fn = os.path.basename(sslfn)
-		m = re.match("(.*)_(certifiate.pem|cert_sign_req.csr|private_key.pem)$", fn)
+		m = re.match("(.*)_(certificate.pem|cert_sign_req.csr|private_key.pem)$", fn)
```

Comment:
⚠️ This needs attention.

Reply:
Yep, that should be reverted. The typo is documented in the code comments around there.

Reply:
Done!